src/apps/lwaftr/alarms.lua

```diff
@@ -0,0 +1,25 @@
+module(..., package.seeall)
```

Comment:
Perhaps place in `program/lwaftr` ?  I mean, it's more about the lwaftr network function as a whole and not the apps.  Dunno tho.